Dark.qss

```diff
@@ -1043,3 +1043,24 @@ AboutDialog QTextEdit {
 ViewCert {
   min-height: 600px;
 }
+
+TalkingUI {
+  background-color: #191919;
+}
+
+TalkingUI > * {
+  background-color: #191919;
+}
+
+TalkingUI [selected=false] {
+  background-color: #191919;
+}
+
+TalkingUI [selected=false]:hover {
+  background-color: #333;
+}
+
+TalkingUI [selected=true] {
+  background-color: #3e4f5e;
+  border: 1px solid #3e4f5e;
+}
```

Comment:
Keep newline at the bottom.

Reply:
This is actually how the preprocessor program thingy generated it... But I can add it manually :)